mm2src/coins/lightning/ln_sql.rs

```diff
@@ -93,25 +93,24 @@ fn insert_channel_sql(for_coin: &str) -> Result<String, SqlError> {
     Ok(sql)
 }
 
-fn upsert_payment_sql(for_coin: &str) -> Result<String, SqlError> {
+fn insert_payment_sql(for_coin: &str) -> Result<String, SqlError> {
     let table_name = payments_history_table(for_coin);
     validate_table_name(&table_name)?;
 
     let sql = format!(
-        "INSERT OR REPLACE INTO {} (
+        "INSERT INTO {} (
             payment_hash,
             destination,
             description,
             preimage,
-            secret,
             amount_msat,
             fee_paid_msat,
             is_outbound,
             status,
             created_at,
             last_updated
         ) VALUES (
-            ?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11
+            ?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10
```

Comment:
How about giving a name to each SQL parameter like it's done for `starts_swaps`:
https://github.com/KomodoPlatform/atomicDEX-API/blob/728b20db56c25eb35a00cc1e922267bb4736923f/mm2src/mm2_main/src/database/stats_swaps.rs#L135-L150